src/Tabs.jsx

```diff
@@ -0,0 +1,180 @@
+import React from 'react';
+import cx from 'classnames';
+
+/**
+ * @module TabsListTab
+ *
+ * Direct child of `TabsList`.
+ * Renders individual tab control.
+ * Accepts a `selected` prop.
+ */
+export const TabsListTab = React.createClass({
```

Comment:
Okay I dug in and at first I was getting the error you're describing but then I realized I wasn't exporting all the classes that I needed to, so once I fixed that everything worked fine. [Here's the commit](https://github.com/meetup/meetup-web-components/commit/c627058) that has everything working using `class` - let me know if you're still having issues after checking that out